e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-panel-filter-remove.spec.js

```diff
@@ -34,7 +34,7 @@ context('Dashboard Deals - panel selected filters - remove a filter', () => {
     dashboardFilters.showHideButton().click();
 
     // apply filter
-    dashboardFilters.panel.form.submissionType.MIA.checkbox().click();
+    dashboardDeals.filters.panel.form.submissionType.MIA.checkbox().click();
     dashboardFilters.panel.form.applyFiltersButton().click();
```

Comment:
should this one be changed?

Reply:
`dashboardDeals`are things specific to deals, `dashboardFilters`is a cypress partial that has generic dashboard filter elements that aren't deals/facilities specific. Do you think it could be made clearer somehow?

Reply:
could it just be filters? dashboardFilters and dashboardDeals.filters looks confusing

Reply:
Thing is `filters` is a generic name and it is specific to dashboard. How about these instead?:

`dashboard.filters`
`dashboardDeals.filters`
`dashboardFacilities.filters`

But then it's pretty much as the same as what you're saying isn't clear. Not sure what else to suggest  😕

Reply:
^ I've done this in the latest commit, I think it's clearer now. The file names remain the same, but the variable name/import is just `filters` so when e.g `deals.filters` and generic filter elements are used it's clearer imo